README.md

```diff
@@ -53,8 +50,7 @@ To contribute a patch, follow these instructions (note that installing
 ```
 
 
-License (see also LICENSE.txt)
-==============================
+## License (see also LICENSE.txt)
```

Comment:
@chrismattmann Sure. But I think you have applied these changes to the Readme.md with the commit [update README.md](https://github.com/apache/oodt/commit/1b42cacacfc96e96b69f0c28d7eefcb89ae85856). Therefore, I will rollback this change